src/main/java/gregtech/integration/theoneprobe/provider/CoverProvider.java

```diff
@@ -0,0 +1,233 @@
+    public String getID() {
+        return GTValues.MODID + ":coverable_provider";
+    }
+
+    @Override
+    protected void addProbeInfo(@Nonnull ICoverable capability, @Nonnull IProbeInfo probeInfo, @Nonnull EntityPlayer player, @Nonnull TileEntity tileEntity, @Nonnull IProbeHitData data) {
+        CoverBehavior coverBehavior = capability.getCoverAtSide(data.getSideHit());
+        if (coverBehavior instanceof CoverConveyor) {
+            conveyorInfo(probeInfo, (CoverConveyor) coverBehavior);
+        } else if (coverBehavior instanceof CoverPump) {
+            pumpInfo(probeInfo, (CoverPump) coverBehavior);
+        } else if (coverBehavior instanceof CoverItemFilter) {
+            itemFilterInfo(probeInfo, (CoverItemFilter) coverBehavior);
+        } else if (coverBehavior instanceof CoverFluidFilter) {
+            fluidFilterInfo(probeInfo, (CoverFluidFilter) coverBehavior);
+        }
+    }
+
+    /**
+     * Displays text for {@link CoverConveyor} related covers
+     *
+     * @param probeInfo the info to add the text to
+     * @param conveyor  the conveyor to get data from
+     */
+    private static void conveyorInfo(@Nonnull IProbeInfo probeInfo, @Nonnull CoverConveyor conveyor) {
+        String rateUnit = " {*cover.conveyor.transfer_rate*}";
+
+        if (conveyor instanceof CoverItemVoiding) {
+            itemVoidingInfo(probeInfo, (CoverItemVoiding) conveyor);
+        }
+
+        // do not display the regular rate if the cover has a specialized rate
+        if (!(conveyor instanceof CoverItemVoiding) && (!(conveyor instanceof CoverRoboticArm) || ((CoverRoboticArm) conveyor).getTransferMode() == TransferMode.TRANSFER_ANY)) {
+            transferRateText(probeInfo, conveyor.getConveyorMode(), rateUnit, conveyor.getTransferRate());
+        }
+
+        ItemFilterContainer filter = conveyor.getItemFilterContainer();
+        if (conveyor instanceof CoverRoboticArm) {
+            CoverRoboticArm roboticArm = (CoverRoboticArm) conveyor;
+            transferModeText(probeInfo, roboticArm.getTransferMode(), rateUnit, filter.getTransferStackSize(), filter.getFilterWrapper().getItemFilter() != null);
+        }
+        itemFilterText(probeInfo, filter.getFilterWrapper().getItemFilter());
+    }
+
+    /**
+     * Displays info for {@link CoverItemVoiding} related covers
+     *
+     * @param probeInfo the info to add the text to
+     * @param voiding   the voiding cover to get data from
+     */
+    private static void itemVoidingInfo(@Nonnull IProbeInfo probeInfo, @Nonnull CoverItemVoiding voiding) {
+        String unit = " {*gregtech.top.unit.items*}";
+
+        ItemFilterContainer container = voiding.getItemFilterContainer();
+        if (voiding instanceof CoverItemVoidingAdvanced) {
+            CoverItemVoidingAdvanced advanced = (CoverItemVoidingAdvanced) voiding;
+            VoidingMode mode = advanced.getVoidingMode();
+            voidingText(probeInfo, mode, unit, container.getTransferStackSize());
```

Comment:
This displays the voiding amount in Overflow mode, even when there is a item filter in the voiding cover, which overrides the amount set by Overflow mode.